src/sentry/static/sentry/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -99,47 +95,109 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
       dropped: string;
       filtered: string;
     };
+    error?: Error;
   } {
-    const {dataCategory} = this.state;
-
-    let sumTotal = 0;
-    let sumAccepted = 0;
-    let sumDropped = 0;
-    let sumFiltered = 0;
-
+    const cardData = {
+      total: '-',
+      accepted: '-',
+      dropped: '-',
+      filtered: '-',
+    };
     const chartData: ChartStats = {
       accepted: [],
       dropped: [],
       projected: [],
     };
 
-    // Please ignore this stub
-    for (let i = 1; i <= 31; i++) {
-      const date = `Mar ${i}`;
-
-      chartData.accepted.push({value: [date, 2000]} as any); // TODO(ts)
-      chartData.dropped.push({value: [date, 1000]} as any); // TODO(ts)
+    if (!orgStats) {
+      return {cardData, chartData};
+    }
 
-      sumTotal += 5000;
-      sumAccepted += 2000;
-      sumDropped += 1000;
-      sumFiltered += 2000;
+    try {
+      const {dataCategory} = this.props;
+      const interval = '1d'; // TODO(org-stats)
+      const dateTimeFormat = interval === '1d' ? 'MMM D' : 'MMM D, HH:mm';
+
+      const usageStats: UsageStat[] = orgStats.intervals.map(i => {
+        const dateTime = moment(i);
+
+        return {
+          date: dateTime.format(dateTimeFormat),
+          total: 0,
+          accepted: 0,
+          filtered: 0,
+          dropped: {total: 0},
+        };
+      });
+
+      orgStats.groups.forEach(group => {
+        const {outcome, category} = group.by;
+        if (category !== dataCategory) {
+          return;
+        }
+
+        const stats = this.mapSeriesToStats(dataCategory, group.series);
+        stats.forEach((s, i) => {
+          usageStats[i][outcome] = outcome === Outcome.DROPPED ? {total: s} : s;
+        });
+      });
+
+      let sumTotal = 0;
+      let sumAccepted = 0;
+      let sumDropped = 0;
+      let sumFiltered = 0;
+      usageStats.forEach(s => {
+        s.total = s.accepted + s.filtered + s.dropped.total;
+
+        // Card Data
+        sumTotal += s.total;
+        sumAccepted += s.accepted;
+        sumDropped += s.dropped.total;
+        sumFiltered += s.filtered;
+
+        // Chart Data
+        chartData.accepted.push({value: [s.date, s.accepted]} as any);
+        chartData.dropped.push({value: [s.date, s.dropped.total]} as any);
+      });
```

Comment:
Can we get rid of the one char name here too? In this case you could destructure to avoid naming it at all.